tests/Aspire.Hosting.Docker.Tests/DockerComposePublisherTests.cs

```diff
@@ -137,6 +137,8 @@ public async Task DockerComposeAppliesServiceCustomizations()
 
         builder.Services.AddSingleton<IResourceContainerImageBuilder, MockImageBuilder>();
 
+        var containerNameParam = builder.AddParameter("param1", "default-name", publishValueAsDefault: true);
```

Comment:
```suggestion
        var containerNameParam = builder.AddParameter("param-1", "default-name", publishValueAsDefault: true);
```

To test the string replace logic.